app/controllers/fhir/r4/api_controller.rb

```diff
@@ -534,8 +554,8 @@ def accept_header?
   end
 
   # Check content type header for correct mime type
-  def content_type_header?
-    request.headers['Content-Type']&.include?('application/fhir+json')
+  def content_type_header?(header)
+    request.headers['Content-Type']&.include?(header)
```

Comment:
I know you were just updating what was here, but I think just a string comparison to `request.content_type` will do here and is a little cleaner.

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/522/commits/b40fde9b0fbaa7a506f463b7b9bfc4dcef649c35.